src/ScottPlot/PlotDimensions.cs

```diff
@@ -14,6 +14,11 @@ public class PlotDimensions
         public readonly float DataHeight;
         public readonly float DataOffsetX;
         public readonly float DataOffsetY;
+        public float TransformScale { get; private set; } = 1;
+        public void SetOutputDimensions(int width, int height)
+        {
+            TransformScale = System.Math.Min(width / Width, height / Height);
```

Comment:
I wonder if it would be more intuitive if there were a `TransformScaleX` and a `TransformScaleY`. I think if the user passes in values which do not preserve the aspect ratio we should not go out of our way to preserve the aspect ratio. Especially if doing so means we create an image of a different size than the user requested.